### PR TITLE
Closes #1103 Add News view to match Content Hub mockups

### LINKED ISSUE #1103: Add News view to match C/A "Content Hub" mockups
## Motivation
Casual Astronaut frequently includes a "Content Hub" section in their mockups. This commonly includes a category filter, custom view title, featured article, and category tags on the news teasers (separate issue).

### Screenshot
image

## Solution
- [ ] Add a new view to News that includes a category filter for existing taxonomy
- [ ] #1500
- [ ] Add new view to News using the Feature display mode
- [ ] Connect the view displays into one /content-hub page

### PR DESCRIPTION
**Pull request by cpoff**
## Updates News view and adds two new display modes to that view:

- Grid with Filter
- Content Hub - Feature

This should solve issue #1103. 

I marked as DRAFT so I can go through some process testing before submitting as a legit PR.

### CODE REVIEW

File: modules/custom/az_news/config/install/core.entity_view_mode.node.feature.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+langcode: en
```
Comment (trackleft) [inline modules/custom/az_news/config/install/core.entity_view_mode.node.feature.yml:1]: This file should probably exist in https://github.com/az-digital/az_quickstart/tree/main/modules/custom/az_core/config/install with the other display modes

File: modules/custom/az_news/config/install/views.view.az_news.yml [hunk 2]
```diff
@@ -549,6 +578,653 @@ display:
         - 'user.node_grants:view'
         - user.permissions
       tags: {  }
+  block_1:
```
Comment (trackleft) [inline modules/custom/az_news/config/install/views.view.az_news.yml:581]: Should add a more descriptive display id like this one: https://github.com/az-digital/az_quickstart/pull/1410/files#diff-0859d416184da2cd73ce4dadd648c939bbb1bdc62c00aa627a026265c124314fL488

File: modules/custom/az_news/config/install/views.view.az_news.yml [hunk 3]
```diff
@@ -549,6 +578,653 @@ display:
...
+      contexts:
+        - 'languages:language_content'
+        - 'languages:language_interface'
+        - url
+        - url.query_args
+        - user
+        - 'user.node_grants:view'
+        - user.permissions
+      tags: {  }
+  block_2:
```
Comment (trackleft) [inline modules/custom/az_news/config/install/views.view.az_news.yml:773]: Should add a more descriptive id like this one https://github.com/az-digital/az_quickstart/pull/1410/files#diff-0859d416184da2cd73ce4dadd648c939bbb1bdc62c00aa627a026265c124314fL488

File: modules/custom/az_news/config/install/views.view.az_news.yml [hunk 4]
```diff
@@ -549,6 +578,653 @@ display:
...
+    display_plugin: block
+    id: block_2
+    display_title: 'Content Hub - Feature'
```
Comment (trackleft) [inline modules/custom/az_news/config/install/views.view.az_news.yml:776]: This one looks a lot like the marquee view display, but maybe not.

**Comment by trackleft** [comment]
Personally, I would name the view displays after the node view mode and some additional context like `paged` or whatever distinguises the display from the others.

**Comment by camikazegreen** [comment]
> Though it would be great if we could all use the same language/terminology for features.

I have regular check-ins with Casual Astronaut and Brand Engagement and can help to get alignment around language.

**Pull request closed without merging**